time/src/assert-equal.ts

```diff
@@ -1,6 +1,6 @@
 import xs, {Stream} from 'xstream';
 import {deepEqual} from 'assert';
-const variableDiff = require('variable-diff');
+import * as variableDiff from 'variable-diff';
```

Comment:
Not sure if this is the same. Let's see if the tests pass

Reply:
Yeah, this should be the same, and the tests pass locally.

```js
const value = require('foo');
import * as value from 'foo';

const value = require('foo').default;
import value from 'foo';

const {value} = require('foo');
import {value} from 'foo';
```

The only reason why this doesn't feel right is because for the longest time both `TypeScript` and `babel` shipped a shim that would allow you to `import value from 'foo'` and have it work both as `require('foo')` and `require('foo').default`, depending on whether the module had `__esmodule` set.

However, they both correctly follow the import spec now, so all is well in the world.

(Not super related to this PR, just thought it might be interesting for anyone reading).

In any case, I agree, let's see what CI says!